desc/objectives/_coils.py

```diff
@@ -1762,6 +1763,194 @@ def compute(self, params_1, params_2=None, constants=None):
         return Psi
 
 
+class LinkingCurrent(_Objective):
+    """Target the self-consistent poloidal linking current between the plasma and coils.
```

Comment:
should we say somewhere that target should be zero?

Reply:
as I could see target being passed in as the linking current if a user did not know any better (rather, if the user did know the source code)